packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.test.tsx

```diff
@@ -26,7 +26,7 @@ const flags: Partial<Flags> = {
   aclpResourceTypeMap: [
     {
       serviceType: 'dbaas',
-      supportedRegionIds: 'us-west, us-east',
+      supportedRegionIds: 'us-west, us-east, ,',
```

Comment:
why we need a change here

Reply:
to test the empty string filtering logic which we just added